src/test/java/de/mediathekview/mserver/crawler/zdf/json/ZdfDayPageDeserializerTest.java

```diff
@@ -137,7 +137,7 @@ public ZdfDayPageDeserializerTest(
                   "https://api.zdf.de/tmd/2/ngplayer_2_3/vod/ptmd/tivi/170717_folge_35_raeumt_auf_nic")
             },
             Optional.of(
-                WireMockTestBase.MOCK_URL_BASE
+                "http://localhost:8589"
```

Comment:
den festen String auf localhost solltest du noch ersetzen durch wireMockServer.baseUrl()

Reply:
Wiremock Server ist in dieser Klasse nicht definiert. Da auch die Expected HTML-Files den Port fix definiert haben, hab ich es wieder raus genommen und fix definiert. Kann angepasst werden wenn die Expected-HTML-Files mit dynamischen Ports angepasst werden.